### ISSUE TITLE
Incorrect Content Window Corner Rendering

### ISSUE THREAD

**Issue by warelik** [artsyfriedchicken/EdgyArc-fr#22]
**Describe the bug**
When visiting certain websites (example provided in screenshot), the content window corner does not display correctly. Instead of a smooth rounded corner, there is a mismatch with a straight angle overlaid by a rounded frame. The content underneath this frame remains visible.

**To Reproduce**
1. Open the website as shown in the provided screenshot.
2. Observe that the corner of the content window does not display as a rounded corner but instead shows a straight angle with a rounded frame over it.

**Expected behavior**
The corners of the content window should be smoothly rounded without any overlaying frames or visible content beneath.

**Screenshots**
<img width="484" alt="Screenshot 2024-04-01 at 17 15 19" src="https://github.com/artsyfriedchicken/EdgyArc-fr/assets/54947489/b25f3b62-5151-49a9-87ff-52a5bd26b2bd">

**Desktop (please complete the following information):**
 - OS: MacOS
 - Browser Firefox
 - Version 124.0.1

**Additional context**
To rectify this issue, CSS modifications were required, specifically removing any code related to overflow: hidden !important; and background-color: transparent !important;.

**Comment by artsyfriedchicken** [comment]
Yes. known bug. happens because of how clipping is implemented in css when backdrop-filter is present on a website. The current "workaround" is a work in progress and is not perfect by any stretch.

**Comment by warelik** [comment]
I'll redo it - it didn't turn out exactly as I originally intended.
